ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+-----|-----------|------------
+status|string|`"success"`
+transaction|string|Transaction envelope XDR, base64 encoded. This transaction will have the original signature(s) from the request as well as the asset issuer's signature.
+message|string|A human readable string containing information to pass on to the user (optional).
+
+#### Rejected Response
+
+A Rejected response will have a `422` HTTP status code and `rejected` as the `status` value. It means that the transaction is not compliant and thus not signed by the server.
+
+The error message should explain why the transaction is not compliant and advice further actions for the client if possible. If there are further actions that the user must complete before this transaction can be approved, the issuer may provide a URL via `action_url` to faciliate the actions. If the issuer needs to asynchronously validate the transaction, the adviced wait time should be included in the error message if possible so the client would know when to retry.
+
+Parameters:
+
+Name | Data Type | Description
+-----|-----------|------------
+status|string|`"rejected"`
+error|string|A human readable string explaining why the transaction is not compliant.
+action_url|string|A URL that allows the user to complete the actions required to have the transaction approved (optional).
+
+## Transaction Composition
+
+Wallets will have to put operations that fully authorize the accounts for the asset, transact the asset, and deauthorize the accounts in a single transaction. In other words, wallets can build transactions where a single operation, which has been approved by the issuer, can be run. If wallets compose transactions this way, there will be no point where accounts have open, authorized trustlines to run unapproved operations because of transaction atomocity.
+
+Depending on whether issuers want to allow the accounts to maintain offers, issuers can leave the accounts in the `AUTHORIZE_TO_MAINTAIN_LIABILITIES_FLAG` state or completely deauthorize the accounts when completing the operation. See [CAP-18] for the fine-grained control of authorization.
+
+Here is an example of the transaction:
+
+```
+Operation 1: Issuer uses AllowTrust to fully authorize account A, asset X
+Operation 2: Issuer uses AllowTrust to fully authorize account B, asset X
+Operation 3: Payment from A to B
+Operation 4: Issuer uses AllowTrust to set account B, asset X to AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG state
+Operation 5: Issuer uses AllowTrust to set account A, asset X to AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG state
+```
+
+## Discussion
+
+### Should my asset be a regulated asset?
+
+Ideally, no. Implementing Regulated Assets should only be used when absolutely necessary, such as for regulatory reasons. It comes with a substantial operational overhead, added complexity, and burden for users. Issuers should only go down this route if it is absolutely required.
+
+### Why doesn’t the approval server submit transactions to the network?
```

Comment:
Eh, I feel like this was an internal question during design, but I'm not sure Wallets are going to be asking this question. Wallets already submit transactions to Stellar (for example, SEP-24 withdrawal payments), so I don't think it requires an explanation.

Reply:
I think this needs to be called out because people have asked this question, who should submit the transaction, and it hasn't always been the wallets.